packages/react/src/examples/forms/validationHybrid.stories.tsx

```diff
@@ -334,12 +359,14 @@ export const Hybrid = () => {
             </div>
             {formik.values.parkingPeriod === 'temporary' && (
               <div className="hds-example-form__item">
-                <TextInput
+                <DateInput
```

Comment:
It probably would be a good idea to add some kind of check to these now that they use actual DateInput. For example, the permit end date should not be in the past. I do not know how hard implementing a such check would be but I think it could be a good example.

Reply:
- [ ] Check that the permit end date should not be in the past

Reply:
Hmm, it seems it does not validate when I change the date by using keyboard 🤔